app/search.py

```diff
@@ -8,19 +8,52 @@
 import webbrowser
 from googlesearch import search
 import sys
+import os
 
 def open_browser(urls):
     for url in urls:
         webbrowser.open_new_tab(url)
 
-def search_my_term(term, count):
-    return [url for url in search(term, stop=count)]
+def search_my_term(term,s, count):
+    return [url for url in search(term, start=s,stop=count)]
 
+def ignore():
+    with open("data/ignore.txt") as f:
+        return f.read().splitlines()
+    
+def required(urls,ignore_list):
+    url_req=[]
+    for url in urls:
+        flag=0
+        for domain in ignore_list:
+            if(url.find(domain)!=-1):
+                flag=1
+                break
+        if(flag==0):
+            url_req.append(url)
+    return url_req
 
 term = sys.argv[1]
 count = int(sys.argv[2])
 
 print (term, count)
 
-urls = search_my_term(term, count)
-open_browser(urls)
+urls=[]
```

Comment:
Why are we initializing them, looks like it would work just find without these statements? Correct me if I am wrong